spec/superspawn.spec.js

```diff
@@ -100,20 +125,20 @@ describe('spawn method', function () {
         let promise;
         expect(() => { promise = superspawn.spawn(TEST_SCRIPT, []); }).not.toThrow();
 
-        return Promise.resolve(promise).then(_ => {
+        return promise.then(() => {
```

Comment:
This will break if superspawn actually does throw

Reply:
I don't understand what you mean by "break".

In case the `superspawn.spawn()` call does throw a synchronous exception, as opposed to rejecting the returned promise, then Jasmine should detect the failure in the line above. I think whatever happens after that should be OK (though the test case should ideally not hang in case of a failure).

Reverting this change would cause the following test failure on Jasmine@latest in my workarea:

```
1) spawn method Test 006 : inavlid cmd script should not throw but reject with error on non-Windows host
  - Unhandled promise rejection: Error: /Users/brodybits/dev/cordova/cordova-common/spec/fixtures/echo-args.cmd: Command failed with exit code EACCES
  Error: /Users/brodybits/dev/cordova/cordova-common/spec/fixtures/echo-args.cmd: Command failed with exit code EACCES
      at ChildProcess.whenDone (/Users/brodybits/dev/cordova/cordova-common/src/superspawn.js:135:23)
      at ChildProcess.emit (events.js:198:13)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:246:12)
      at onErrorNT (internal/child_process.js:415:16)
      at process._tickCallback (internal/process/next_tick.js:63:19)
```

I don't know how to explain this 100% right, suspect it has something to do with the way a minor Jasmine update interacts with the behavior of our `superspawn`.

Reply:
You are right. My bad. Sorry for the confusion.